### PR TITLE
TSPS-142 Add wdl/dockerfile/ for stdpopsim tool

### PR DESCRIPTION
**Pull request by jsotobroad**
### Description 

stdpopsim is the tool we are currently using to generate fake reference panel data.  we wanted to wrap the tool in a docker so that will make it easier to run in the future.

Successful run of this wdl workflow for 10k samples - https://bvdp-saturn-dev.appspot.com/#workspaces/tsps_dev_bp_02_01_2024_v1/imputation-pipeline-testing-03-07-2024/workflows-app/submission-monitoring/178d57b6-fd77-436b-b29f-4e05215189e9/a682d5d3-a44c-4baf-a43a-9db15af14496

successrul of service launched workflow using the new 1000G reference path - https://bvdp-saturn-dev.appspot.com/#workspaces/tsps_dev_bp_04_22_2024_protect/TSPS_imputation_control_workspace_realWDL_20240521/workflows-app/submission-history/31b1c4fb-986b-4d66-a4ec-ec341c257e69

### Jira Ticket
https://broadworkbench.atlassian.net/browse/TSPS-142

### CODE REVIEW

**Review by mmorgantaylor — APPROVED** [review]
sweet

File: pipelines/simulatedData/StdPopSim.wdl [hunk 1]
```diff
@@ -0,0 +1,58 @@
+version 1.0
+
+workflow StdPopSim {
+
+    input {
+        Int numEuropeanSamples
```
[the remark this answers is not in the document]
Comment (jsotobroad) [reply pipelines/simulatedData/StdPopSim.wdl:6]: yea for a scale test this is probably the two ends of the spectrum so thats why i chose these. there are other demographic files we could use to get other populations.  I dont think we really care all that much for scale testing as you suggested

File: pipelines/testing/ImputationBeagleEmpty.wdl [hunk 2]
```diff
@@ -13,7 +13,7 @@ workflow ImputationBeagle {
 
         File ref_dict # for reheadering / adding contig lengths in the header of the output VCF, and calculating contig lengths
         Array[String] contigs
-        String reference_panel_path # path to the bucket where the reference panel files are stored for all contigs
+        String reference_panel_path_prefix # path to the bucket where the reference panel files are stored for all contigs
```
Comment (mmorgantaylor) [inline pipelines/testing/ImputationBeagleEmpty.wdl:16]: super minor: should we update the comment here, or just delete all the comments and leave the explanations on the main wdl?

**Review by mmorgantaylor — APPROVED** [review]
great!

File: pipelines/testing/ImputationBeagleEmpty.wdl (hunk 2 above)
Comment (jsotobroad) [reply pipelines/testing/ImputationBeagleEmpty.wdl:16]: good point, i think i'll just delete all the comments

**Review by mmorgantaylor — APPROVED** [review]
doubly approved!

**Pull request merged**